auth-web/src/components/pay/UnlinkedShortNameTable.vue

```diff
@@ -0,0 +1,448 @@
+        if (response?.data) {
+          /* We use appendToResults for infinite scroll, so we keep the existing results. */
+          state.results = appendToResults ? state.results.concat(response.data.items) : response.data.items
+          state.totalResults = response.data.total
+          emit('shortname-state-total', response.data.stateTotal)
+        } else {
+          throw new Error('No response from getEFTShortNames')
+        }
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error('Failed to getEFTShortNames list.', error)
+      }
+      state.loading = false
+    }
+
+    function openAccountLinkingDialog (item: any) {
+      accountLinkingDialog.value.open()
+    }
+
+    function closeAccountLinkingDialog () {
+      accountLinkingDialog.value.close()
+    }
+
+    async function clearFilters () {
+      clearFiltersTrigger.value++
+      dateRangeReset.value++
+      state.filters.filterPayload = {}
+      state.filters.isActive = false
+      await loadTableData()
+    }
+
+    function updateFilter (filterField?: string, value?: any) {
+      if (filterField) {
+        if (value) {
+          state.filters.filterPayload[filterField] = value
+          state.filters.isActive = true
+        } else {
+          delete state.filters.filterPayload[filterField]
+        }
+      }
+      if (Object.keys(state.filters.filterPayload).length === 0) {
+        state.filters.isActive = false
+      } else {
+        state.filters.isActive = true
+      }
+    }
+
+    async function infiniteScrollCallback () {
+      state.filters.pageNumber++
+      await loadTableData(null, null, true)
+    }
+
+    return {
+      actionDropdown,
+      clearFilters,
+      clearFiltersTrigger,
+      infiniteScrollCallback,
+      headers,
+      tableDataOptions,
+      state,
+      title,
+      updateFilter,
+      formatAmount,
+      formatDate,
+      updateDateRange,
+      showDatePicker,
+      dateRangeReset,
+      clickDatePicker,
+      dateRangeSelected,
+      accountLinkingDialog,
+      openAccountLinkingDialog,
+      closeAccountLinkingDialog,
+      datePicker
+    }
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/scss/theme.scss';
+
+#linked-bank-short-names {
+  border: 1px solid #e9ecef
+}
+
+// For the dropdown text color.
+::v-deep .theme--light.v-list-item .v-list-item__action-text, .theme--light.v-list-item .v-list-item__subtitle {
+  color: $app-blue;
+  font-weight: normal;
+  .v-icon.v-icon {
+    color: $app-blue;
+  }
+}
+
+.new-actions {
+  height:30px;
+  .open-action-btn {
+    font-size: .875rem;
+    box-shadow: none;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    margin-right: 1px;
+  }
+
+  .more-actions-btn {
+    box-shadow: none;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+}
+
+::v-deep {
+
+  .base-table__header > tr:first-child > th  {
```

Comment:
Same with here possibly, I'll put it into a SCSS file we can share across our components